src/components/SideNavbar.tsx

```diff
@@ -40,7 +40,8 @@ export interface NavItems {
 interface navItemProp {
   name: string;
   route: string;
-  currentRoute: string;
+  activeRoute: string;
```

Comment:
This `activeRoute` prop is unnecessary, we can just use `useLocation` when we want the current browser path.

Reply:
you're right, i'll include that in the navbar refactoring branch